src/modules/40-gitsync/pages/config/GitFullSyncEntityList.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import type { CellProps, Column, Renderer } from 'react-table'
+import { Color, Icon, Layout, TableV2, Text } from '@harness/uicore'
+import type { IconProps } from '@harness/uicore/dist/icons/Icon'
+import { useStrings } from 'framework/strings'
+import type { PageGitFullSyncEntityInfoDTO, GitFullSyncEntityInfoDTO } from 'services/cd-ng'
+import { getEntityIconName } from '../entities/EntityHelper'
+import css from './GitSyncConfigTab.module.scss'
+
+interface GitFullSyncEntityListProps {
+  data: PageGitFullSyncEntityInfoDTO
+  gotoPage: (pageNumber: number) => void
+}
+
+interface StatusData {
+  icontProp: IconProps
+  statusBackground?: Color
+  highlightedColor?: Color
+}
+
+const RenderEntityStatus = (prop: Record<'status', GitFullSyncEntityInfoDTO['syncStatus']>): JSX.Element => {
+  const status = prop.status
+  let data: StatusData
+
+  switch (prop.status) {
+    case 'FAILED':
+      data = {
+        icontProp: { name: 'warning-sign', color: Color.RED_900 },
+        statusBackground: Color.RED_100,
+        highlightedColor: Color.RED_900
+      }
+      break
+    case 'SUCCESS':
+      data = {
+        icontProp: { name: 'success-tick' },
+        statusBackground: Color.GREEN_100,
+        highlightedColor: Color.GREEN_700
+      }
+      break
+    case 'QUEUED':
+      data = {
+        icontProp: { name: 'queued' },
+        statusBackground: Color.GREY_100,
+        highlightedColor: Color.GREY_900
+      }
+      break
+    default:
+      data = { icontProp: { name: 'placeholder' } }
+  }
+
+  const { icontProp, statusBackground, highlightedColor } = data
+
+  return (
+    <Layout.Horizontal
+      spacing="small"
+      background={statusBackground}
+      padding={{ top: 'small', right: 'small', bottom: 'small', left: 'small' }}
```

Comment:
I think you can just pass `small` instead of specifying each side
```suggestion
      padding="small"
```